lib/rules/fluent-chaining.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * @fileoverview Enforce formatting for chained calls
+ */
+
+'use strict';
+
+var RuleHelper = require('../helpers/rule-helper');
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+module.exports = {
+   create: function(context) {
+      var helper = new RuleHelper(context);
+
+      //--------------------------------------------------------------------------
+      // Helpers
+      //--------------------------------------------------------------------------
+
+      function validateSingleLineMemberExpression(node) {
+         if (node.object.loc.start.line !== node.property.loc.start.line) {
+            context.report({
+               node: node.property,
+               message: 'Identifier {{ identifier }} should be on a new line',
+               data: {
+                  identifier: node.property.name
+               },
+               fix: function(fixer) {
+                  var firstLineIndent = helper.lineIndent(node.object.loc.end.line),
+                      fixString = '\n' + helper.indentChar.repeat(firstLineIndent) + '.';
+
+                  return fixer.replaceTextRange([ node.object.end, node.property.start ], fixString);
+               }
+            });
+         }
+      }
+
+      function validateMultiLineMemberExpression(node) {
+         var firstLineIndent = helper.lineIndent(node.object.loc.start.line),
+             lastLineIndent = helper.lineIndent(node.object.loc.end.line),
+             expectedIndent = firstLineIndent + helper.indent,
+             propertyLineIndent = helper.lineIndent(node.property.loc.start.line);
+
+         if (propertyLineIndent !== expectedIndent) {
+            context.report({
+               node: node.property,
+               message: 'Identifier {{ identifier }} should be indented {{ expectedIndent }} spaces',
```

Comment:
The previous note on this line did not get resolved. It said:

> I don't think this message is clear. If I read it I think of "indented" as a verb - meaning "this many more spaces", rather than "this many total spaces".